README.md

```diff
@@ -390,3 +392,10 @@ There are two way to add custom devices types.
 	If you have a brand new device-type, it is the simpliest way.
 * Copy a patch addding/modifying a device-type in lava-master/device-types-patch/
 	If you are modifying an already present (upstream) device-type, it is the best way.
+
+## How to made LAVA slave use a proxy ?
+Add env to a slave like:
+slave:
+  env:
+  - "overrides:"
+  - "  http_proxy: http://dns:port"
```

Comment:
Why the leading spaces?

This looks like more than just setting environment variables, but also setting a specific syntax required by the LAVA config file.  Can you remove the requirement for the user to know about the syntax/whitespace?